scripts/Inelastic/IndirectImport.py

```diff
@@ -49,13 +50,26 @@ def _os_env():
     return platform.system() + platform.architecture()[0]
 
 
-def _lib_suffix():
-    if platform.system() == "Windows":
-        suffix = "win"
-    elif platform.system() == "Linux":
-        suffix = "lnx"
+def is_pip_version_of_libs():
+    name = 'Quest'
```

Comment:
Can I just check my understanding here? We pick an arbitrary library, `Quest`, and if it's already imported or is discoverable then it looks like we have the `pip` versions because the old ones have the `_lnx` or `_win` suffixes?

If so could you had a comment to that effect in order to make it clear what's expected to happen?